src/test/scala/no/ndla/articleapi/UnitSuite.scala

```diff
@@ -22,9 +22,12 @@ abstract class UnitSuite extends FunSuite with Matchers with OptionValues with I
   setEnv("SEARCH_REGION", "some-region")
   setEnv("RUN_WITH_SIGNED_SEARCH_REQUESTS", "false")
 
-  setEnv("MIGRATION_HOST", "some-host")
-  setEnv("MIGRATION_USER", "some-user")
-  setEnv("MIGRATION_PASSWORD", "some-password")
+  setEnv("AUDIO_API_URL", "localhost:30014")
```

Comment:
Det stemmer. Man får en feilmelding, men den er ikke veldig tydelig. (bare en import exception som sier at artikkelen ikke kunne importeres). Legger inn noen sjekker før testene kjører for å sjekke at alle dependencies er på plass.

Jeg ser også at det ikke er noen enkel måte å se _hvilken_ artikkel som evt. failer - jeg fikser så det blir tydeligere også